package.json

```diff
@@ -22,13 +22,14 @@
   "author": "GoDaddy Operating Company, LLC",
   "license": "MIT",
   "engines": {
-    "node": ">=8.0.0"
+    "node": ">=12.0.0"
```

Comment:
while node@8 hasn't been LTS for a while.  _technically_ this is a breaking change here instead of a minor one.

Reply:
Fine to release as a new major.